src/types.ts

```diff
@@ -0,0 +1,168 @@
+import { ScEventType } from "./scEvent";
+import { ScLinkContentType, TContentString } from "./ScLinkContent";
+
+export interface IContentResult {
+  value: number | string | null;
+  type: TContentString;
+}
+
+interface ISetContentPayload {
+  command: "set";
+  type: TContentString;
+  data: string | number;
+  addr: number;
+}
+
+interface IGetContentPayload {
+  command: "get";
+  addr: number;
+}
+
+interface IFindKeynode {
+  command: "find";
+  idtf: string;
+}
+
+interface IResolveKeynode {
+  command: "resolve";
+  idtf: string;
+  elType: number;
+}
+
+type TKeynodesPayload = Array<IFindKeynode | IResolveKeynode>;
+
+export interface INode {
+  el: "node";
+  type: number;
+}
+
+export interface IEdgeSide {
```

Comment:
We usually call it like `IEdgeInfo` so it would probably be better name here. Please update references and utils `transformEdgeSide` method name